packages/notifi-react-card/lib/components/subscription/subscription-card-views/HistoryCardView.tsx

```diff
@@ -23,49 +27,72 @@ import {
   AlertNotificationRow,
   AlertNotificationViewProps,
 } from '../../AlertHistory/AlertNotificationRow';
-import { LoadingStateCard, LoadingStateCardProps } from '../../common';
+import NotifiAlertBox, {
+  NotifiAlertBoxButtonProps,
+  NotifiAlertBoxProps,
+} from '../../NotifiAlertBox';
+import { SignupBanner, SignupBannerProps } from '../../SignupBanner';
+import { VerifyBanner, VerifyBannerProps } from '../../VerifyBanner';
+import { LoadingStateCardProps } from '../../common';
+import Spinner from '../../common/Spinner';
 
 export type NotificationHistoryEntry =
   | Types.FusionNotificationHistoryEntryFragmentFragment
   | Types.NotificationHistoryEntryFragmentFragment;
 
 export type AlertHistoryViewProps = Readonly<{
   noAlertDescription?: string;
-  isHidden: boolean;
   data: CardConfigItemV1;
   copy?: {
+    /**
+     * @deprecated No longer have loading header when loading history
+     */
     loadingHeader?: string;
+    /**
+     * @deprecated No longer have loading content when loading history
+     */
     loadingContent?: string;
     loadingSpinnerSize?: string;
     loadingRingColor?: string;
   };
   classNames?: DeepPartialReadonly<{
-    title: string;
-    header: string;
-    manageAlertLink: string;
-    noAlertDescription: string;
-    notificationDate: string;
-    notificationSubject: string;
-    notificationMessage: string;
-    notificationImage: string;
-    notificationList: string;
-    emptyAlertsBellIcon: string;
-    historyContainer: string;
-    virtuoso: string;
-    AlertCard: AlertNotificationViewProps['classNames'];
-    LoadingStateCard: LoadingStateCardProps['classNames'];
+    title?: string;
+    header?: string;
+    NotifiAlertBox?: NotifiAlertBoxProps['classNames'];
+    dividerLine?: string;
+    manageAlertLink?: string;
+    noAlertDescription?: string;
+    notificationDate?: string;
+    notificationSubject?: string;
+    notificationMessage?: string;
+    notificationImage?: string;
+    notificationList?: string;
+    emptyAlertsBellIcon?: string;
+    historyContainer?: string;
+    virtuoso?: string;
+    AlertCard?: AlertNotificationViewProps['classNames'];
+    /**
+     * @deprecated Use `loadingSpinner` instead
+     */
+    LoadingStateCard?: LoadingStateCardProps['classNames'];
```

Comment:
let's remove this! people that upgrade SDK will need to change their styling anyway